### PR TITLE
Update RTCD docs to reflect that we use Deployments now

### PR DESCRIPTION
**Pull request by d-wierdsma**
#### Summary

Update RTCD docs to reflect that we use Deployments now

#### Ticket Link

https://mattermost.atlassian.net/browse/CLD-5537

### CODE REVIEW

**Review by andrleite — APPROVED** [review]

**Comment by streamer45** [comment]
Thanks @d-wierdsma! For my own understanding, is there any practical difference in this change from an application perspective? I assume no, but I just want to make sure that the ability to scale horizontally as designed (DNS based load balancing) won't be affected.

**Comment by d-wierdsma** [comment]
> Thanks @d-wierdsma! For my own understanding, is there any practical difference in this change from an application perspective? I assume no, but I just want to make sure that the ability to scale horizontally as designed (DNS based load balancing) won't be affected.

No changes in how the service operates at all, this change is purely to prevent downtime during Kubernetes cluster upgrades as we can not freely roll daemonsets as easily as deployments.

File: source/configure/calls-deployment.rst [hunk 1]
```diff
@@ -176,7 +176,9 @@ An example with sample values:
 
   securityContext: {}
 
-  daemonset:
+  deployment:
+    replicas: 2
```
[the remark this answers is not in the document]
Comment (d-wierdsma) [reply source/configure/calls-deployment.rst:180]: Nope so this is 2 replicas total, if you take a look at the `affinity` section it ensures that the two replicas are hosted on two separate nodes, preferably in different regions.

This number comes from the number of Calls nodes that we typically run in cloud, which is 2, if a user has more nodes they can increase the number of replicas. The exposure of the port and host will be the exact same with these deployments as they utilize the hosts network and opens up the port there.

FWIW this is already running in dev, test, and community without issues.

[the remark this answers is not in the document]
Comment (streamer45) [reply source/configure/calls-deployment.rst:180]: > if you take a look at the affinity section it ensures that the two replicas are hosted on two separate nodes

Thanks, that's what I was looking for.

**Review by streamer45 — APPROVED** [review]
Thanks :100:

**Review by stylianosrigas — APPROVED** [review]

**Comment by streamer45** [comment]
@d-wierdsma Is this waiting for more updates or can be merged?

**Pull request merged**